.github/workflows/mingw.yml

```diff
@@ -0,0 +1,54 @@
+name: MinGW
+on:
+  pull_request:
+    branches: [ '*' ]
+  push:
+    branches: [ '*' ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+jobs:
+  mingw:
+    if: github.repository == 'aws/aws-lc'
```

Comment:
Should we add this to our other jobs? Is there a reason not to let people run this CI on there forks?

Reply:
Isn't it only expensive if they open PR's to their own repo? Which raises the question why is this running on pushes?